src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -201,6 +201,137 @@ When the Scheduler is restarted, all the AWS instances that were previously crea
 
 
 
+===== OpenStack Infrastructure
+
+To use the OpenStack instances as computing nodes, a specific OpenStack Infrastructure
+can be created using the Resource Manager. This section describes briefly how to make it.
+
+1.  First, you need to have an admin account on your OpenStack server
```

Comment:
Link to OpenStack documentation?

http://docs.openstack.org/icehouse/install-guide/install/yum/content/keystone-users.html